### ISSUE TITLE
[BE] Mock Database in Tests

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#74]
Mock Database Adapter PyMongo in the tests and include the skipped tests again

## Acceptance criteria
- [ ] Tests run successfully in the GitHub Workflow

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
